pandas/tests/frame/test_constructors.py

```diff
@@ -2396,6 +2396,19 @@ def check_views():
             # assert b[0] == 0
             assert df.iloc[0, 2] == 0
 
+    def test_consistency_of_string_columns_with_none(self):
+        # df created from list, None is casted to str
+        df = DataFrame(["1", "2", None], columns=["a"], dtype="str")
+
+        assert isinstance(df.loc[0].values[0], str)
```

Comment:
```suggestion
        assert isinstance(df.loc[0, "a"], str)
```

Reply:
Yeah, seems better, I think the issue was like that so it got carried.